scripts/fix-joined-kind-clusters.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
As per the CI failure, you'll need to add the boilerplate text to the top of this file (with 2019 as the year):

https://github.com/kubernetes-sigs/federation-v2/blob/master/hack/boilerplate.go.txt